cloud/aws/.goreleaser.yaml

```diff
@@ -0,0 +1,40 @@
+---
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
```

Comment:
Are these comments left over from a template?

Reply:
this was copied over from the CLI, so will need to remove them there as well.